templates/patterns/card/card.ui_patterns.yml

```diff
@@ -1,119 +1,66 @@
 card:
-  label: "Card"
-  description: "A card is a flexible and extensible content container. It includes options for headers and footers, a wide variety of content, contextual background colors, and powerful display options. https://v5.getbootstrap.com/docs/5.0/components/card"
+  label: 'Card'
+  description: 'A card is a flexible and extensible content container. It includes options for headers and footers, a wide variety of content, contextual background colors, and powerful display options. For more information: https://v5.getbootstrap.com/docs/5.0/components/card'
   variants:
     default:
-      label: Default
-    horizontal:
-      label: Horizontal
+      label: 'Default'
+      description: 'Standard bootstrap card.'
     search:
-      label: Search
-  settings:
-    style:
-      type: select
-      label: Card style
-      options:
-        primary: primary
-        secondary: secondary
-        success: success
-        danger: danger
-        warning: warning
-        info: info
-        light: light
-        dark: dark
-    border_variant:
-      type: select
-      label: Card border style
-      options:
-        primary: primary
-        secondary: secondary
-        success: success
-        danger: danger
-        warning: warning
-        info: info
-        light: light
-        dark: dark
-    text_color:
-      type: select
-      label: Card text color
-      options:
-        dark: dark
-        white: white
-    horizontal:
-      type: boolean
-      label: "Horizontal"
-      description: "Horizontal card"
-      preview: false
+      label: 'Search'
+      description: 'Card with special behaviour for print search results items.'
   fields:
+    orientation:
+      type: string
+      label:  'Orientation'
+      description: 'Can be set to horizontal or vertical. Defaults to vertical.'
     title:
-      type: "array"
-      label: "Title"
-      description: "Card title [content, tag, classes]"
-      preview:
-        content: "Title card"
-        tag: "h5"
+      type: text
+      label: 'Title'
+      description: 'Card title.'
+      preview: 'Title card'
     subtitle:
-      type: "array"
-      label: "Subtitle"
-      description: "Card subtitle [content, tag, classes]"
-      preview:
-        content: "Subtitle card"
-        classes: "mb-2"
+      type: text
+      label: 'Subtitle'
+      description: 'Card subtitle.'
+      preview: 'Subtitle card'
     text:
-      type: "array"
-      label: "Text"
-      description: "Card text [content, tag, classes]"
-      preview:
-        content: "Text card with longer text which is supported"
-        classes: "mb-2"
+      type: text
+      label: 'Text'
+      description: 'Card text.'
+      preview: 'Text card with longer text which is supported'
     image:
```

Comment:
I've noticed this `oe_bootstrap_theme_preprocess_pattern_card` but I don't think we need it anymore.